internal/bridge/crunchybridgecluster/crunchybridgecluster_controller.go

```diff
@@ -466,24 +518,23 @@ func (r *CrunchyBridgeClusterReconciler) reconcileBridgeConnectionSecret(
 	if err != nil {
 		meta.SetStatusCondition(&crunchybridgecluster.Status.Conditions, metav1.Condition{
 			ObservedGeneration: crunchybridgecluster.GetGeneration(),
-			Type:               v1beta1.ConditionCreating,
-			Status:             metav1.ConditionFalse,
+			Type:               v1beta1.ConditionReady,
+			Status:             metav1.ConditionUnknown,
 			Reason:             "SecretInvalid",
 			Message: fmt.Sprintf(
-				"Cannot create with bad secret: %v", "TODO(crunchybridgecluster)"),
+				"Cannot create with bad secret: %v", err),
```

Comment:
This is no longer a "creating" condition so let's reword this message to be more generic.